docs/dataformats/observation_lists.rst

```diff
@@ -10,38 +10,87 @@ the information in the run list. These filenames can either be given directly
 or there can be naming conventions, e.g. the event list for run ``42`` could be stored
 at ``$TEV_DATA/RUN_000042/Events.fits``.
 
+A run list must have at least a column called ``OBS_ID``::
+ 
+   OBS_ID
+   42
+   43
 
-CSV format
-----------
+Usually it has many more columns with information about each observation. A list of Gammapy supported columns is:
 
-We use the `CSV <http://en.wikipedia.org/wiki/Comma-separated_values>`_ (comma-separated values) format for run lists.
-This has the advantage that everyone can work with whatever tool they like. Here's some good options:
+================  ================================================================================================  =========
+column name       description                                                                                       required?
+================  ================================================================================================  =========
+OBS_ID            observation ID as an integer                                                                      yes
+RA                pointing position right ascension in equatorial (ICRS) coordinates                                yes?
+DEC               pointing position declination in equatorial (ICRS) coordinates                                    yes?
+AZ                average azimuth angle during the observation                                                      yes?
+ALT               average altitude angle during the observation                                                     yes?
+MUON_EFFICIENCY   average muon efficiency of the telescopes                                                         yes?
+TIME_START        start time of the observation stored as number of seconds after the time reference in the header  yes?
```

Comment:
OK, that's what I thought. So we don't make any suggestions about the origin in our doc. I was confused because of the reference to the Fermi website. Now I see, the website is meant only for the MET description, but not for the MET origin. I could add a sentence: each experiment will have a different MET origin that should be included in the header of the data files. What do you think?